### ISSUE TITLE
Remove GZ ignore rule from .gitignore

### ISSUE THREAD

**Issue by grok** [pantheon-systems/WordPress#141]
There is a core file that is constantly getting dropped from installs because of: `*.gz` as an ignore rule.

Typically lives at `web/wp-includes/js/tinymce/wp-tinymce.js.gz` or `wp-includes/js/tinymce/wp-tinymce.js.gz`.

Noticing that it's lost frequently when you transfer the file to a webroot structure (unless you do a proper `git mv`) and it's lost during core updates and upstream updates.

I don't think there's much a savings by having that ignore rule, and since it's causing at least one core file to go missing, I think it should be dropped.

**Comment by greg-1-anderson** [comment]
Drupal 8 has a similar issue; the drops-8 .gitignore solves this problem by removing the restriction for just the core directory:
```
# Things in the core directory that Drupal 8 commits in the repository.
!core/**/*.gz
```
We could do something similar for Wordpress:
```
# Things in wp-includes directory that WordPress commits in the repository.
!wp-includes/**/*.gz
```

**Comment by pwtyler** [comment]
Looks like this was shipped in #176 (two years ago!) but never closed.
